expression/evaluator_test.go

```diff
@@ -353,23 +316,31 @@ func (s *testEvaluatorSuite) TestBinopBitop(c *C) {
 		{nil, ast.RightShift, 1, nil},
 	}
 
-	for _, t := range tbl {
-		fc := funcs[t.op]
-		f, err := fc.getFunction(s.ctx, datumsToConstants(types.MakeDatums(t.lhs, t.rhs)))
-		c.Assert(err, IsNil)
+	ctx := mock.NewContext()
+	ctx.GetSessionVars().StmtCtx.TimeZone = time.Local
+	sc := ctx.GetSessionVars().StmtCtx
+	sc.TruncateAsWarning = true
+	err := ctx.GetSessionVars().SetSystemVar("max_allowed_packet", "67108864")
+	require.NoError(t, err)
+	ctx.GetSessionVars().PlanColumnID = 0
```

Comment:
You may factor out these lines to a utility function `createEvaluatorSuite` or something.

Reply:
I find that ```createContext``` can init context instead of this code block.

Reply:
Yep. I create it to supersede `SetUpTest` when migrating builtin tests 😉